cmd/ci-operator/main.go

```diff
@@ -294,6 +297,8 @@ func bindOptions(flag *flag.FlagSet) *options {
 
 	flag.StringVar(&opt.kubeconfig, "kubeconfig", "", "Path to .kube/config file. First config whose host matches the cluster is used to access imagestreamtags. If not set or no matching config , use the anonymous user")
 
+	flag.StringVar(&opt.pullSecretPath, "pull-secret-path", "", "A path of a docker config file.")
```

Comment:
Let's label this with what it is used for rather than what it is

```
flag.StringVar(&opt.pullSecretPath, "image-import-pull-secret", "", "A set of dockercfg credentials used to import images for the tag_specification.")
```

Reply:
Done.